docs/kb/task_manager.rst

```diff
@@ -0,0 +1,93 @@
+
+There are two types of tasks: *cluster* and *node* tasks. Node tasks cover operations that are performed on a single
+node. To see their stats, you need to request a particular node. Cluster tasks are responsible for the operations
+that spread across many nodes. They are visible from all nodes in the cluster. Cluster tasks can't have parents. They
+can have children on many nodes. The status of a cluster task's child may not be accessible even though the cluster
+operation is still running.
+
+A task may be *internal*. This means that the task has a parent or is started by an internal process. By default API
+calls skip the internal tasks. Cluster tasks cannot be internal.
+
+If a task is internal, it is unregistered from task manager immediately after it is finished. If it has a non-cluster
+parent, the task's status is folded into its parent and accessible only through the parent. You won't see the statuses
+of indirect descendants (e.g. children of children) of a finished task unless they have failed.
+
+Other non-cluster tasks stay in task manager for *task_ttl* second after they are finished. task_ttl value can be set
+with ``task_ttl_in_seconds`` param or through ``/task_manager/ttl`` api. The time for which cluster tasks are visible
+in task manager isn't specified.
+
+
+Task manager API
+----------------
+
+Data structures
+^^^^^^^^^^^^^^^
+
+Task data, that is returned from the task manager API, is kept in either ``task_stats`` or ``task_status``.
+
+task_stats
+..........
+
+- *task_id* - unique task id;
+- *type* - a type of the task, e.g. offstrategy compaction;
+- *kind* - whether the task is per node or cluster;
+- *scope* - specifies the operation's scope, e.g. keyspace, range;
+- *state* - one of created, running, done, failed;
+- *sequence_number* - an operation number (per module). It is shared by all tasks in a tree. Irrelevant for cluster tasks;
+- *keyspace* - optional, name of a keyspace on which the task works;
+- *table* - optional, name of a table on which the task works;
+- *entity* - optional, additional info specific to the task.
+
+
+task_status
+...........
+
+All filed from task_stats and additionally:
+- *is_abortable* - a flag that decides whether the task can be aborted through API;
+- *start_time* - relevant only if state == created;
+- *end_time* - relevant only if the task is finished (state in [done, failed]);
+- *error* - relevant only if the task failed;
+- *parent_id* - relevant only if the task has a parent;
+- *shard* - optional, number of a shard on which the task works;
+- *progress_units* - a unit of progress;
+- *progress_total* - job size in progress_units;
+- *progress_completed* - current progress in progress_units;
+- *children_ids* - list of pairs of children ids and nodes on which they are created.
+
+API calls
+^^^^^^^^^^
+
+`/task_manager/list_modules` - lists modules supported by task manager;
+`/task_manager/list_module_tasks/{module}` - lists tasks in the module
```

Comment:
The above two are not part of the list, and formatting is off in the rendered docs.

Also, I think you need to enclose these in two '``' to enforce formatting as code.